frontend/javascripts/oxalis/view/action-bar/volume_actions_view.js

```diff
@@ -1,84 +1,280 @@
-          style={{ marginRight: 10 }}
+        <img src="/assets/images/overwrite-empty.svg" alt="Overwrite Empty Icon" />
+      </RadioButtonWithTooltip>
+    </Radio.Group>
+  );
+}
+
+export default function VolumeActionsView() {
+  const hasSkeleton = useSelector(state => state.tracing.skeleton != null);
+  const activeTool = useSelector(state => enforceVolumeTracing(state.tracing).activeTool);
+  const activeCellId = useSelector(state => enforceVolumeTracing(state.tracing).activeCellId);
+  const isMappingEnabled = useSelector(
+    state => state.temporaryConfiguration.activeMapping.isMappingEnabled,
+  );
+  const isInMergerMode = useSelector(state => state.temporaryConfiguration.isMergerModeEnabled);
+  const mappingColors = useSelector(
+    state => state.temporaryConfiguration.activeMapping.mappingColors,
+  );
+
+  useEffect(() => {
+    if (isInMergerMode) {
+      Store.dispatch(setToolAction(VolumeToolEnum.MOVE));
+    }
+  }, [isInMergerMode, activeTool]);
+
+  const isShiftPressed = useKeyPress("Shift");
+  const isControlPressed = useKeyPress("Control");
+  const isAltPressed = useKeyPress("Alt");
+
+  const customColors = isMappingEnabled ? mappingColors : null;
+  const activeCellColor = convertCellIdToCSS(activeCellId, customColors);
+
+  const adaptedActiveTool = adaptActiveToolToShortcuts(
+    activeTool,
+    isShiftPressed,
+    isControlPressed,
+    isAltPressed,
+  );
+
+  const moveToolHint = hasSkeleton
+    ? getMoveToolHint(activeTool, isShiftPressed, isControlPressed, isAltPressed)
+    : null;
+  const previousMoveToolHint = usePrevious(moveToolHint);
+
+  const disabledVolumeExplanation =
+    "Volume annotation is disabled while the merger mode is active.";
+
+  const moveToolDescription = `Pointer – Use left-click to move around${
+    hasSkeleton ? " and right-click to create new skeleton nodes" : ""
+  }.`;
+
+  return (
+    <div
+      onClick={() => {
+        if (document.activeElement) document.activeElement.blur();
+      }}
+    >
+      <Radio.Group onChange={handleSetTool} value={adaptedActiveTool}>
+        <RadioButtonWithTooltip
+          title={moveToolDescription}
+          disabledTitle=""
+          disabled={false}
+          style={narrowButtonStyle}
+          value={VolumeToolEnum.MOVE}
         >
-          <RadioButton value={VolumeToolEnum.MOVE}>Move</RadioButton>
-          <Tooltip
-            title={
-              this.props.isInMergerMode
-                ? "Volume annotation is disabled while the merger mode is active."
-                : ""
-            }
-          >
-            <RadioButton value={VolumeToolEnum.TRACE} disabled={this.props.isInMergerMode}>
-              Trace
-            </RadioButton>
-            <RadioButton value={VolumeToolEnum.BRUSH} disabled={this.props.isInMergerMode}>
-              Brush
-            </RadioButton>
+          <Tooltip title={moveToolHint || previousMoveToolHint} visible={moveToolHint != null}>
```

Comment:
Why is the `previousMoveToolHint` relevant here?

Reply:
Good point, I forgot to add an comment for this workaround. I'm adding it now:

```
          {/*
            When visible changes to false, the tooltip fades out in an animation. However, moveToolHint
            will be null, too, which means the tooltip text would immediately change to an empty string.
            To avoid this, we fallback to previousMoveToolHint.
          */}
          <Tooltip title={moveToolHint || previousMoveToolHint} visible={moveToolHint != null}>
```